sdk/python/foundation-models/system/content-safety/deploy_with_content_safety.ipynb

```diff
@@ -0,0 +1,803 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# How to create an Safety-Enabled LLaMa Online Endpoint\n",
+    "## This notebook will walk you through the steps to create a Safety-Enabled LLaMa Online Endpoint.\n",
+    "### The steps are:\n",
+    "1. Create a Content Safety for moderating the request from user and response from the LLaMa Online Endpoint\n",
+    "2. Create a new LLaMa Online Endpoint\n",
+    "3. Create a new Safety-Enabled LLaMa Online Endpoint with a custom score.py file which will be used to moderate the request and response"
```

Comment:
Are we going to create 2 endpoints?
I think Ke has an idea how to only need 1 endpoint, where we re-package the model as another mlflow model. The downside is we have an extra registered model, but upside is user only needs to track / pay for 1 endpoint.

Reply:
Thank you for pointing this out. If merging into 1 endpoint, will customer aware how much is paying for the content safety feature? We could discuss this offline.

Reply:
merged into 1 endpoint.